test_utils/src/lib.rs

```diff
@@ -936,6 +938,274 @@ pub fn mock_signed_certificate(
             .unwrap();
         votes.push((pk.clone(), sig))
     }
-    let cert = Certificate::new(&committee(None), header, votes).unwrap();
+    let cert = Certificate::new(committee, header, votes).unwrap();
     (cert.digest(), cert)
 }
+
+use rand::rngs::OsRng;
+
+pub struct Builder<R = OsRng> {
+    rng: R,
+    committee_size: NonZeroUsize,
+    number_of_workers: NonZeroUsize,
+    randomize_ports: bool,
```

Comment:
this reads like a function name to me, it's more verbose but maybe consider "should_randomize_ports"? or feel free to ignore :)

Reply:
I don't have a strong preference but do lean slightly towards the shorter name 🤷🏼‍♂️